inclusive-design/wecount.inclusivedesign.ca
# feat: finish migrating learn page resources from Airtable

## Description

This pull request adds new Markdown files in the `resources` collection in addition to some changes to others. This also represents the final migration data pull from the Airtable base for this colleciton, and as such the scripts and code which were added in #496 are no longer needed and have been removed.

## Steps to test

1. Run `npm run build`.
2. Run `npm start`
3. Test content editing in the CMS.

**Expected behavior:** The Resources collection should be editable via the CMS.

## Additional information

N/A

## Related issues

#496
